File: Autobus.java
public class Autobus {
   private float kilometraje;
   private String color;

   public float getKilometraje() {
      return kilometraje;
   }

   public String getColor() {
      return color;
   }

   public void setKilometraje(float kilometraje) {
      this.kilometraje = kilometraje;
   }

   public void setColor(String color) {
      this.color = color;
   }

   public static void main(String[] args) {
      Autobus norteSur = new Autobus();

      float kilometraje = norteSur.getKilometraje();
      String color = norteSur.getColor();
      System.out.println("El kilometraje es:" + kilometraje);
      System.out.println("El color es: " + color);
   }
}

File: Python/Autobus.py
class Autobus():
    __kilometraje = None
    __color = None

    def getKilometraje(self):
        return self.__kilometraje

    def getColor(self):
        return self.__color

    def setKilometraje(self, kilometraje):
        self.__kilometraje = kilometraje

    def setColor(self, color):
        self.__color = color


def main():
    miAutobus = Autobus()
    print("El kilometraje es: ", miAutobus.getKilometraje())
    print("El color es: ")
